### ISSUE TITLE
Adding stdlib documentation to the hakaru documentation

### ISSUE THREAD

**Issue by mkhattab940** [hakaru-CS4ZP6/hakaru#25]
It would be good to create a new section on github.hakaru-dev.io for stdlib documentations.

Documentations should contain brief description of the distribution and its parameters, along with a link to the wikipedia page for the distribution.

The comments included in stdlib files should be a good start

**Comment by JacquesCarette** [comment]
Yes please!

**Comment by mkhattab940** [comment]
@JacquesCarette I have noticed that the Primitive Distributions documentation on the site includes the return type. We don't have any explicit return types in the stdlib so far, but it shouldn't be too hard to add them in and then include that in our documentation.

I am concerned about certain distributions like chiSq() and exponential() which currently return real values, but are actually strictly positive. Should we update these implementations to cast to prob values? Would a change like this effect the tests we've already implemented involving such distributions?

**Comment by JacquesCarette** [comment]
Yes, please make sure all of these return `prob`. There should be no need to cast, as it should be obvious that they are all positive.  It may have a minor effect on the tests (i.e. slight changes in syntax may cause them to fail, but none of the changes should be 'deep').
